class/defaults.yml

```
@@ -2,3 +2,112 @@ parameters:
   topolvm:
     =_metadata: {}
     namespace: syn-topolvm
+
+    deviceclasses:
+      - name: ssd
+        volumegroup: vgssd
+        sparegb: 1
+
+    storageclasses:
+      - name: ssd-local
+        fstype: xfs
+        isdefault: false
+        volumeexpansion: true
```

Same here (default storage class managed by component `storageclass`)

```suggestion
    storageclasses:
      ssd-local:
        fstype: xfs
        device_class: ssd
        allowVolumeExpansion: true
```